src/client/nav/request.js

```diff
@@ -114,6 +114,9 @@ spf.nav.request.send = function(url, opt_options) {
     // the cache response synchronously.
     if (spf.config.get('experimental-sync-response-cache')) {
       handleCache();
+    } else if (spf.config.get('experimental-sync-response-cache-background') &&
+               document.webkitVisibilityState == 'hidden') {
```

Comment:
Is this the same as document.hidden?

Ref: https://developer.mozilla.org/en-US/docs/Web/Guide/User_experience/Using_the_Page_Visibility_API#Methods_overview